app/src/main/java/org/oppia/app/walkthrough/WalkthroughPageChangeListener.kt

```diff
@@ -0,0 +1,5 @@
+package org.oppia.app.walkthrough
+
+interface WalkthroughPageChangeListener {
+  fun pageNumber(pageNo: Int)
+}
```

Comment:
EOF is missing, add one blank line at the end.

Reply:
done